### PR TITLE
Update URLs to point at JuliaVTK org where applicable

### PR DESCRIPTION
**Pull request by sloede**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 
 With **ReadVTK.jl** you can read in data from VTK XML files in Julia. It aims to complement
-the excellent package [WriteVTK.jl](https://github.com/jipolanco/WriteVTK.jl).
-ReadVTK is part of the [Trixi Framework](https://trixi-framework.github.io).
+the excellent package [WriteVTK.jl](https://github.com/JuliaVTK/WriteVTK.jl).
+ReadVTK is part of the [Trixi Framework](https://JuliaVTK.github.io).
 
 *Note: ReadVTK was mainly motivated by wanting to write proper tests for
-[Trixi2Vtk.jl](https://github.com/trixi-framework/Trixi2Vtk.jl).
+[Trixi2Vtk.jl](https://github.com/JuliaVTK/Trixi2Vtk.jl).
```
Comment (ranocha) [inline README.md:15]:
```suggestion
[Trixi2Vtk.jl](https://github.com/trixi-framework/Trixi2Vtk.jl).
```

File: README.md [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
+[![Build Status](https://github.com/JuliaVTK/ReadVTK.jl/workflows/CI/badge.svg)](https://github.com/JuliaVTK/ReadVTK.jl/actions?query=workflow%3ACI)
+[![Coveralls](https://coveralls.io/repos/github/JuliaVTK/ReadVTK.jl/badge.svg?branch=main)](https://coveralls.io/github/JuliaVTK/ReadVTK.jl?branch=main)
 [![License: MIT](https://img.shields.io/badge/License-MIT-success.svg)](https://opensource.org/licenses/MIT)
 <!-- [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.5221552.svg)](https://doi.org/10.5281/zenodo.5221552) -->
 
 With **ReadVTK.jl** you can read in data from VTK XML files in Julia. It aims to complement
-the excellent package [WriteVTK.jl](https://github.com/jipolanco/WriteVTK.jl).
-ReadVTK is part of the [Trixi Framework](https://trixi-framework.github.io).
+the excellent package [WriteVTK.jl](https://github.com/JuliaVTK/WriteVTK.jl).
+ReadVTK is part of the [Trixi Framework](https://JuliaVTK.github.io).
```
Comment (ranocha) [inline README.md:12]:
```suggestion
ReadVTK is part of the [JuliaVTK Framework](https://github.com/JuliaVTK).
```
We don't have a dedicated website (yet).

File: README.md [hunk 3]
```diff
@@ -102,7 +102,7 @@ Helpful resources for working with (i.e., reading and writing) VTK XML files:
 
 ## Authors
 ReadVTK is maintained by the
-[Trixi authors](https://github.com/trixi-framework/Trixi.jl/blob/main/AUTHORS.md).
+[Trixi authors](https://github.com/JuliaVTK/Trixi.jl/blob/main/AUTHORS.md).
```
Comment (ranocha) [inline README.md:105]:
```suggestion
[Trixi authors](https://github.com/trixi-framework/Trixi.jl/blob/main/AUTHORS.md).
```

File: README.md [hunk 4]
```diff
@@ -121,11 +121,11 @@ ReadVTK is licensed under the MIT license (see [LICENSE.md](LICENSE.md)).
 Since ReadVTK is an open-source project, we are very happy to accept contributions
 from the community. Please refer to [CONTRIBUTING.md](CONTRIBUTING.md) for more details. To get in
 touch with the developers, join us on Trixi's
-[Slack workspace](https://join.slack.com/t/trixi-framework/shared_invite/zt-sgkc6ppw-6OXJqZAD5SPjBYqLd8MU~g)
+[Slack workspace](https://join.slack.com/t/JuliaVTK/shared_invite/zt-sgkc6ppw-6OXJqZAD5SPjBYqLd8MU~g)
```
Comment (ranocha) [inline README.md:124]:
```suggestion
[Slack workspace](https://join.slack.com/t/trixi-framework/shared_invite/zt-sgkc6ppw-6OXJqZAD5SPjBYqLd8MU~g)
```

File: docs/src/index.md [hunk 5]
```diff
@@ -1,12 +1,11 @@
...
-With [ReadVTK.jl](https://github.com/trixi-framework/ReadVTK.jl) you can
+With [ReadVTK.jl](https://github.com/JuliaVTK/ReadVTK.jl) you can
 read in data from VTK XML files in Julia. It aims to complement
-the excellent package [WriteVTK.jl](https://github.com/jipolanco/WriteVTK.jl).
-ReadVTK is part of the [Trixi Framework](https://trixi-framework.github.io).
+the excellent package [WriteVTK.jl](https://github.com/JuliaVTK/WriteVTK.jl).
 
 *Note: ReadVTK was mainly motivated by wanting to write proper tests for
-[Trixi2Vtk.jl](https://github.com/trixi-framework/Trixi2Vtk.jl).
+[Trixi2Vtk.jl](https://github.com/JuliaVTK/Trixi2Vtk.jl).
```
Comment (ranocha) [inline docs/src/index.md:8]:
```suggestion
[Trixi2Vtk.jl](https://github.com/trixi-framework/Trixi2Vtk.jl).
```

File: docs/src/index.md [hunk 6]
```diff
@@ -95,7 +94,7 @@ Helpful resources for working with (i.e., reading and writing) VTK XML files:
 
 ## Authors
 ReadVTK is maintained by the
-[Trixi authors](https://github.com/trixi-framework/Trixi.jl/blob/main/AUTHORS.md).
+[Trixi authors](https://github.com/JuliaVTK/Trixi.jl/blob/main/AUTHORS.md).
```
Comment (ranocha) [inline docs/src/index.md:97]:
```suggestion
[Trixi authors](https://github.com/trixi-framework/Trixi.jl/blob/main/AUTHORS.md).
```

**Review by ranocha — CHANGES_REQUESTED** [review]

**Comment by sloede** [comment]
Thanks! I could've sworn I'd have caught all these, but some apparently slipped through :-/

**Review by ranocha — APPROVED** [review]
Thanks!

**Comment by sloede** [comment]
Windows tests passed 🥳

**Comment by sloede** [comment]
I guess now GH org, new luck 😬

**Pull request merged**